integration/solid/dyn_direct_solve/dyn_direct_solve_serial.json

```diff
@@ -1,7 +1,7 @@
 
 {
   "user_name": "meemee",
-  "host_name": "quartz135",
+  "host_name": "quartz46",
```

Comment:
This exposes a potentially annoying problem.  `user_name` and `host_name` will often change.  Either we need to move this out of this file or add an option to not have uniquely identifying info in the summary file for tests.